### DISCUSSION TITLE
Find the author and source

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#688] [Galaxy Trivia] [answered]
> "One thing was certain, that the white kitten had had nothing to do with it: — it was the black kitten’s fault entirely."

**Comment by peterbaccaro** [comment]
The author is **Lewis Carroll**.
The name of the book is **Through the Looking-Glass, and What Alice Found There** (also known as **Alice Through the Looking-Glass** or simply **Through the Looking-Glass**).

**Comment by thuongtruong1009** [comment, marked as answer]
The author of this quote is Lewis Carroll, and it's from his book "Through the Looking-Glass, and What Alice Found There". This quote is spoken by the character of the White Queen in chapter 6, during a conversation with Alice about the trial of the Knave of Hearts. The context of the quote is that the White Queen is trying to explain to Alice that sometimes things happen without any apparent cause or reason, and that it's best to just accept them and move on.
